### PR TITLE
feature(prepayments)

### PR DESCRIPTION
**Pull request by lukedawilson**

### CODE REVIEW

File: Examples/BookingSystem.AspNetCore/Feeds/SessionsFeeds.cs [hunk 1]
```diff
@@ -136,7 +137,8 @@ x.Modified < (DateTimeOffset.UtcNow - new TimeSpan(0, 0, 2)).UtcTicks)
                                     OpenBookingFlowRequirement = result.Item1.RequiresApproval 
                                         ? new List<OpenBookingFlowRequirement> { OpenBookingFlowRequirement.OpenBookingApproval }
                                         : null,
-                                    ValidFromBeforeStartDate = result.Item1.ValidFromBeforeStartDate
+                                    ValidFromBeforeStartDate = result.Item1.ValidFromBeforeStartDate,
+                                    Prepayment = result.Item1.Prepayment.Convert()
```
Comment (nickevansuk) [inline Examples/BookingSystem.AspNetCore/Feeds/SessionsFeeds.cs:141]: Great solution 👍

File: Fakes/OpenActive.FakeDatabase.NET/FakeBookingSystem.cs [hunk 2]
```diff
@@ -889,14 +904,15 @@ public static void CreateSellers(IDbConnection db)
             }
         }
 
-        public (int, int) AddFacility(
+        public (int, int) AddFacility( // ToDo: update switch statement
```
Comment (nickevansuk) [inline Fakes/OpenActive.FakeDatabase.NET/FakeBookingSystem.cs:907]: Is this still outstanding?

**Review by nickevansuk — COMMENTED** [review]
Looks good, assuming tests pass!

File: Fakes/OpenActive.FakeDatabase.NET/FakeBookingSystem.cs (hunk 2 above)
Comment (lukedawilson) [reply Fakes/OpenActive.FakeDatabase.NET/FakeBookingSystem.cs:907]: Oops... No.

**Review by nickevansuk — APPROVED** [review]
Still looks good!

**Pull request merged**